mage/query-modules/python/union-find.md

```diff
@@ -0,0 +1,132 @@
+---
+id: union-find
+title: union_find
+sidebar_label: union_find
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
```

Comment:
Before merging, please update `mage/overview.md` file as you did in README of implementation :smile:

Reply:
Not sure if I understand this entirely :slightly_smiling_face: 
If you take a look at the `mage/overview.md` file under Files changed, you can see that I made some changes to it.

Reply:
My bad, was looking in outdated version of this PR :smile: